### ISSUE TITLE
RC Test: Build An Atom: 1.2.0-rc.1

### ISSUE THREAD

**Issue by aadish** [phetsims/tasks#577]
Build An Atom 1.2.0-rc.1 is ready for testing 

**Link to sim:** http://www.colorado.edu/physics/phet/dev/html/build-an-atom/1.2.0-rc.1/build-an-atom_en.html

**Link to iFrame:** http://www.colorado.edu/physics/phet/dev/html/build-an-atom/1.2.0-rc.1/build-an-atom_en-iframe.html

**Test Matrix:** https://docs.google.com/spreadsheets/d/1ftWkWzaQrRVf_R02LrsONwmmtUMoOKOos2e5l2verXs/edit#gid=2

Please do a full retest of this sim.  This is the first release candidate with support for internationalization, so please use ?stringTest appropriately.  If you notice any issues that are not listed in https://github.com/phetsims/build-an-atom/issues, please create a new issue and reference this one.

**Redeploy General Tests** (copied from https://github.com/phetsims/tasks/issues/528 )
The following tests should also be performed as part of this testing
- [x] stringTest=double
- [x] stringTest=long
- [x] stringTest=X (short strings)
- [x] stringTest=rtl (right-to-left)
- [x] stringTest=xss (should not redirect, OK to look bad, test on one desktop platform)
- [x] showPointerAreas (touchArea=red, mouseArea=blue)
- [x] Full screen test
- [x] Reference issues to RC test task

Please make sure following open issues are tested on this RC
- [x] Game should retain most recent score https://github.com/phetsims/build-an-atom/issues/122
- [x] User report screen turns black https://github.com/phetsims/build-an-atom/issues/76
- [x] Safari: Everything disappears https://github.com/phetsims/build-an-atom/issues/63 

@ariel-phet please make sure I've provided all the necessary details, prioritize and reassign.

**Comment by ariel-phet** [comment]
@phet-steele @elisemorgan @brroberts1231 @BryceAG @bryo5363 please make this the next testing priority after #576

**Comment by ariel-phet** [comment]
Bumping to current top QA priority

**Comment by phet-steele** [comment]
iPad2 iOS 8.4.1 is temporarily unavailable to test on. @ariel-phet says testing on it is not needed for redeploy.

**Comment by phet-steele** [comment]
@aadish @ariel-phet @jbphet RC done.

**Comment by jbphet** [comment]
Thanks everyone!  Closing this, will work with @aadish to create the next RC.
